app/models/Document.ts

```diff
@@ -429,6 +429,20 @@ export default class Document extends ArchivableModel {
     return this.title || i18n.t("Untitled");
   }
 
+  @computed
+  get membershipType(): "collection" | "direct" | "group" {
```

Comment:
Rename `direct` to `document`? Or something different maybe..?

Reply:
`document` is good